### DISCUSSION TITLE
Add field info and/or field notes within icon

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3028] [Feature Requests]
All Field Label info (eg: Saves instantly) should be moved to an info icon with tooltip. These icons would default to gray/subdued, but could also be yellow/warning or even red/danger as needed.

This will allow us to have longer text without making the design ugly/overwhelming. The main issue is that a note will not be immediately visible without hovering over this icon... easy to miss.

![Screen Shot 2020-08-28 at 11 16 52 AM](https://user-images.githubusercontent.com/522079/91584005-00363800-e920-11ea-9edd-9aaec45eca7f.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
